packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -163,7 +178,7 @@ export const getServerSideProps = async function getServerSideProps(
 
     return {
       redirect: {
-        destination: getAbsoluteEventTypeRedirectUrl({
+        destination: getAbsoluteEventTypeRedirectUrlWithEmbedSupport({
```

Comment:
Either headless router(this module) or routing-link needs this method to ensure correct URL is generated.